### DISCUSSION TITLE
Youtube cookies?

### DISCUSSION THREAD

**Discussion by thehijacker** [karakeep-app/karakeep#402] [Q&A]
Hello,

I added few youtube links but they are not correctly indexed as the browser shows a cookie confirmwation page first:

![image](https://github.com/user-attachments/assets/d7714e1e-b571-4de0-81dd-242c3e3aab40)

![image](https://github.com/user-attachments/assets/9570fe53-cbd4-4d25-87d8-43b24a1cdd63)

![image](https://github.com/user-attachments/assets/becd820f-bfb4-4bec-9a31-63911da90c36)

How to bypass this? Thank you.

**Comment by MohamedBassem** [comment]
Yeah, youtube has been a hit and miss for me. Sometimes it works well, and sometimes I hit this cookie banner. I don't have a solution for now, but I'll think about it. Maybe we can special case youtube by auto accepting the cookie banner or something.

**Comment by bearded-papa** [comment]
although I prefer Hoarder I have run both Hoarder and Readeck side by side and Readeck manages to get the webpage from behind the cookie banner without issue so this is possible but is just a difference in code I'm guessing
